src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,53 @@
+import { ErrorType } from '../../types/ErorTypes';
+
+type Props = {
+  error: ErrorType | null;
+  onError: (errors: boolean) => void;
+};
+
+export const Errors: React.FC<Props> = ({ error, onError }) => {
+  const getErrorMessage = () => {
+    switch (error) {
+      case ErrorType.LoadError:
+        return 'Unable to load todos';
+
+      case ErrorType.TitleError:
+        return 'Title should not be empty';
+
+      case ErrorType.AddError:
+        return 'Unable to add a todo';
+
+      case ErrorType.DeleteError:
+        return 'Unable to delete a todo';
+
+      case ErrorType.UpdateError:
+        return 'Unable to update a todo';
+
+      default:
+        return null;
+    }
+  };
+
+  const handleHideMessage = () => {
+    onError(false);
+  };
+
+  return (
+    error ? (
```

Comment:
If you want to hide an error notification it's enough to add a class hidden. Also, I don't see where you hide the error after 3 seconds
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/63243136/58b74b6b-23c0-44e5-ae7f-db576dc59d77)